### PR TITLE
Add to requirements.txt and README

### LINKED ISSUE #5: Make parser a stand-alone package
The parser should be usable on its own outside of hydrus.
We need to find the cleanest way of porting the history in this repository and then build an interface (CLI) to make the parser usable as a stand-alone or library installable via `pip` from hydrus.

### PR DESCRIPTION
**Pull request by de-sh**
Fixes #5 and #10

### Checklist
- [x] My branch is up-to-date with upstream/develop branch.
- [x] Everything works and tested for Python 3.5.2 and above.

### Description

This is to further continue the work done in #9 by completing the process  of moving the parser out of the hydrus repo.

### Change logs
Update README and add to requirements.txt

### CODE REVIEW

**Review by xadahiya — APPROVED** [review]
Looks good! Nice work @de-sh

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+setup(
+    name='hydra_openapi_parser',
+    version='0.1',
+    packages=find_packages(),
+    license='MIT',
+    description='A Parser from OpenAPI to Hydra specs',
+    long_description=open('README.md').read(),
+    long_description_content_type="text/markdown",
+    url='https://github.com/HTTP-APIs/hydra_openapi_parser',
```
Comment (vddesai1871) [inline setup.py]: In url replace underscores with dashes.

**Review by vddesai1871 — COMMENTED** [review]
Looks good

**Comment by de-sh** [comment]
Guys, shouldn't we be following the same structure in the setup.py file as in the other repos?

Also, when I used BFG-repo-cleaner to delete a file from history, all the older commit hashes also changed, and that is why they show up here. Anyways, only changes made, can be [viewed here](https://github.com/HTTP-APIs/hydra-openapi-parser/pull/11/files).

**Pull request merged**